README.md

```diff
@@ -22,8 +20,7 @@ Requires:
   - xmltodict
```

Comment:
Can we just remove this requirements section? At this point it's incomplete and inaccurate since we no longer require `shapely`. Also, we mention below in the "Environment Setup" that users need to run pip install for the packages which should take care of this section for us.

Reply:
This is already being changed in CV-96 brach